bitnami/redis/templates/scripts-configmap.yaml

```diff
@@ -74,37 +74,56 @@ data:
 
     validate_quorum() {
         if is_boolean_yes "$REDIS_TLS_ENABLED"; then
-            quorum_info_command="redis-cli {{- if and .Values.auth.enabled .Values.auth.sentinel }} -a $REDIS_PASSWORD {{- end }} -h $REDIS_SERVICE -p $SENTINEL_SERVICE_PORT --tls --cert ${REDIS_TLS_CERT_FILE} --key ${REDIS_TLS_KEY_FILE} --cacert ${REDIS_TLS_CA_FILE} sentinel master {{ .Values.sentinel.masterSet }}"
+            quorum_info_command="redis-cli -h $REDIS_SERVICE -p $SENTINEL_SERVICE_PORT --tls --cert ${REDIS_TLS_CERT_FILE} --key ${REDIS_TLS_KEY_FILE} --cacert ${REDIS_TLS_CA_FILE} sentinel master {{ .Values.sentinel.masterSet }}"
```

Comment:
Hi,
  Instead of setting and unsetting the `REDISCLI_ATUH`, what about this ?
```suggestion
            quorum_info_command="{{- if and .Values.auth.enabled .Values.auth.sentinel }}REDISCLI_AUTH="$REDIS_PASSWORD" {{- end }}redis-cli -h $REDIS_SERVICE -p $SENTINEL_SERVICE_PORT --tls --cert ${REDIS_TLS_CERT_FILE} --key ${REDIS_TLS_KEY_FILE} --cacert ${REDIS_TLS_CA_FILE} sentinel master {{ .Values.sentinel.masterSet }}"
```

Reply:
@rafariossaa I tried this, but it only works with eval, not with subshells. It gave me:

```
my-release-redis-node-1 sentinel /opt/bitnami/scripts/start-scripts/start-sentinel.sh: line 82: REDISCLI_AUTH=thepass: command not found
```

I switched the subshells to eval, which did indeed work. However, We're now back to the original issue of the password being logged.

We may be able to change `REDISCLI_AUTH="$REDIS_PASSWORD"` to `REDISCLI_AUTH="\$REDIS_PASSWORD"` to avoid the password being inlined while the command is generated.

This requires `REDIS_PASSWORD` to be exported, which in my testing was the case.

So that ends up with:

```
my-release-redis-node-2 sentinel  12:48:53.46 INFO  ==> about to run the command: REDISCLI_AUTH=$REDIS_PASSWORD redis-cli -h my-release-redis.default.svc.cluster.local -p 26379 sentinel get-master-addr-by-name mymaster
```

It's a bit more fragile, but more concise and consistent as well.

Reply:
Pushed a change that does this, please take another look!